pom.xml

```diff
@@ -71,6 +71,7 @@
         <java.level>8</java.level>
         <useBeta>true</useBeta>
         <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
+        <hpi.compatibleSinceVersion>2.40</hpi.compatibleSinceVersion>
```

Comment:
Not strictly correct but the best we can do to encourage admins to read the changelog and understand the impact of upgrading.